device-types/Extreme Networks/X460-G2-48x-10GE4.yaml

```diff
@@ -0,0 +1,186 @@
+    mgmt_only: false
+  - name: '7'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '8'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '9'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '10'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '11'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '12'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '13'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '14'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '15'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '16'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '17'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '18'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '19'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '20'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '21'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '22'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '23'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '24'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '25'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '26'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '27'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '28'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '29'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '30'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '31'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '32'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '33'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '34'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '35'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '36'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '37'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '38'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '39'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '40'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '41'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '42'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '43'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '44'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '45'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '46'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '47'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '48'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '49'
+    type: 10gbase-x-sfpp
+    mgmt_only: false
+  - name: '50'
+    type: 10gbase-x-sfpp
+    mgmt_only: false
+  - name: '51'
+    type: 10gbase-x-sfpp
+    mgmt_only: false
+  - name: '52'
+    type: 10gbase-x-sfpp
+    mgmt_only: false
+  - name: '53'
+    type: other
```

Comment:
Which do these default as? They should be defined such as their defaults are. We try to not use other if possible